applications/launchpad/gui-react/src/components/CoinsList/index.tsx

```diff
@@ -6,12 +6,13 @@ import { CoinsListProps } from './types'
 
 const formatAmount = (amount: string) => {
   if (Number(amount) === 0) {
-    return '00 000'
+    return '00,000'
```

Comment:
you could do something like `Number(amount).toLocaleString([], { minimumFractionDigits: 3 })`
it would give you the 3 0s in fraction, because now if you have locale that uses `.` as fraction separator only "0,000" will have `,`

Reply:
Thanks! not sure if we need fractions here. I suppose that we will work with _integers_ in mining. So I'm going to skip this for now

Reply:
ah, I thought `,` is a fraction separator, not thousand separator, make sense then